ngsild-payloads/applicationPod.jsonld

```diff
@@ -0,0 +1,89 @@
+{
+  "id": "urn:ngsi-ld:ApplicationPod:istio-ingressgateway-9c8b9b586-thl7x",
+  "type": "ApplicationPod",
+  "name": {
+    "type": "Property",
+    "value": "istio-ingressgateway-9c8b9b586-thl7x"
+  },
+  "temporalResolution": {
+    "type": "Property",
+    "value": "PT1H"
+  },
+  "cpuUtilization": [
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:00:00Z",
+      "value": 0.06907686666721323,
+      "unitCode": "CORE"
+    },
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:05:00Z",
+      "value": 0.07907686666721322,
+      "unitCode": "CORE"
+    }
+  ],
+  "networkTransmission": [
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:00:00Z",
+      "value": 196.1,
+      "unitCode": "B"
+    },
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:05:00Z",
+      "value": 200.1,
+      "unitCode": "B"
+    }
+  ],
```

Comment:
you can't have more than one instance of a property with the default dataset (unless you are in a temporal representation, which is typically not the case in the samples). here, I think you can remove one of the two.